lib/konbata/models/upload_course.rb

```diff
@@ -85,6 +103,12 @@ def upload_content(filename)
       puts "Uploading: #{name}"
       upload_to_s3(migration, filename)
       puts "Done uploading: #{name}"
+
+      if File.exist?(source_for_imscc)
```

Comment:
That sounds good. For now, I think this works. Let's not worry about changing it.